src/main/resources/localization/messages-pl.properties

```diff
@@ -0,0 +1,78 @@
+error.generic.delete = Giveaway został usunięty, ale wystąpił problem podczas usuwania jego wiadomości.
+error.generic.creation = Wystapił problem podczas tworzenia giveawaya.
+error.generic.enter = Wystąpił problem podczas dołączania do giveawaya.
+error.generic.reroll = Wystapił problem podczas ponownego losowania zwycięzców giveawaya.
+error.generic = Wystąpił problem.
+warning.no_giveaways = Obecnie nie ma żadnych aktywnych giveawayi!
+warning.not_supported = Ta funkcja nie jest obecnie wspierana!
+warning.no_entries = Brak ważnych uczestników, więc wygrany nie mógł zostać wylosowany!
+success.giveaway_created = Giveaway został pomyślnie stworzony! ID: {0}
+success.entered = Pomyślnie dołączyłeś do giveawaya!
+success.leave = Pomyślnie opuściłeś giveawaya!
+success.ping = Pong!
+success.winner = Gratulacje {0}! Wygrałeś **{1}**!
+success.settings.role = Użytkownicy z rolą {0} mogą teraz zarządzać giveawayami!
+success.settings.color = Kolor emebda dla giveawayi został zmieniony na `{0}`.
+success.settings.emoji = Emotka customowego przycisku została zmieniona na `{0}`.
+success.giveaway_ended = Pomyślnie zakończono giveawaya {0}!
+success.giveaway_reroll = {0} ponownie wylosował zwycięzców giveawaya, gratulacje {1}!
+success.giveaway_delete = Pomyślnie usunięto gievawaya {0}!
+info.invite = Witaj! Jestem **GiveawayBot**! Pomagam stworzyć giveawaye w prosty i przystępny sposób! \
+\nMożesz dodać mnie do swojego serwera za pomocą tego linku:
+info.about = Wszystko na temat
+info.about.brief = Przeprowadzaj giveawaye szybko i prosto!
+info.about.long = Witaj! Jestem **GiveawayBot**, i jestem tutaj po to aby sprawić, że tworzenie giveawayów na twoim serwerze Discord stanie się proste i przyjemne! Zostałem stworzony przez [**jagrosh**#4824](https://jagrosh.com/) (<@113156185389092864>). Wszystkie moje komendy zaczynają się na `/g`. Nie zapomnijcie obczaić mojej strony dostepnej pod tym linkiem -> **https://giveawaybot.party/**!
+info.about.stats = Statystyki
+info.about.stats.servers = {0} serwerów
+info.about.stats.giveaways = {0} giveawayów
+info.about.links = Linki
+info.about.links.website = Strona
+info.about.links.invite = Zaproszenie
+info.about.links.support = Pomoc
+info.settings = ustawienia
+info.settings.owner = Właściciel serwera
+info.settings.premium = Poziom premium
+info.settings.emoji = Emtka giveawayi
+info.settings.role = Rola zarządcy giveawayów
+info.settings.color = Kolor embedów
+info.settings.locale = Język
+info.settings.etc = Aby kontrolować która rola zarządza giveawayami, udaj się do zakładki **Server Settings** > **Integrations** > **GiveawayBot**
+info.help.general = Komendy ogólne
+info.help.creation = Komendy tworzenia giveawayi
+info.help.manipulation = Komendy manipulacji giveawayami
+info.help.settings = Komendy ustawień GiveawayBota
+giveaway.leave = Opuść Giveawaya
+giveaway.ends = Kończy się
+giveaway.ended = Zakończył się
+giveaway.hosted = Hostowany przez
+giveaway.entries = Wejść
```

Comment:
Change the word to "Wpisy"

Reply:
Afaik this is used in the context of the ammount of entries, in that case "xxx wejść" sounds better than "xxx wpisów" as "wejść" is more commonly used in the context of giveaways.

Reply:
In the case of only 1 entries or an amount of entries ending with 2 neither make sense but to fix that we would need 2 more conig options